packages/components/src/theme/panel.scss

```diff
@@ -0,0 +1,41 @@
+.panel-content {
+    @include panel-variant($panel-content-border, $panel-content-text, $panel-content-heading-bg, $panel-content-border);
+
+    & > .panel-heading {
+        border-bottom: 1px solid #ddd;
```

Comment:
do we have a scss variable for this `#ddd` color?

Reply:
Fixed.